server/index.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const http = require('http');
+const db = require('../database/index');
+
+
+const app = express();
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+
+app.use((req, res, next) => {
+  res.header('Access-Control-Allow-Origin', '*');
+  res.header('Access-Control-Allow-Headers', 'Origin, X-Requested-With, Content-Type, Accept');
+  next();
+});
+
+app.use(express.static('public'));
+
+app.get('/favicon.ico', (req, res) => res.status(204));
+
+const server = http.createServer(app);
```

Comment:
Is there a reason you're using http module even though you're already using express? Could just do app.listen(3003)

Reply:
Ya, I remembered that Beth told us that express by itself cannot handle socket, http one can do that.